packages/dev/core/src/Engines/webgpuEngine.ts

```diff
@@ -3220,7 +3223,7 @@ export class WebGPUEngine extends Engine {
 
         this._rttRenderPassWrapper.colorAttachmentViewDescriptor = {
             format: this._colorFormat as GPUTextureFormat,
-            dimension: WebGPUConstants.TextureViewDimension.E2d,
+            dimension: texture.is3D ? WebGPUConstants.TextureViewDimension.E3d : WebGPUConstants.TextureViewDimension.E2d,

```

Comment:
This was hardcoded to 2D before. Does this seem like the appropriate change?

Reply:
pinging @Popov72